deployments/karnot/deployment-config.yaml

```diff
@@ -50,3 +50,29 @@ environments:
           MAX_FELTS_IN_CALLDATA: "22500"
           MONGO_REPLACE_DATA_INSIDE_TRANSACTION: "true"
```

Comment:
thanks for the PR, can you just update the KAKAROT_ADDRESS values?
KAKAROT_ADDRESS: "0x15626fc5070f7675966aac4f3a4dc5a9c721cd8df311c6d05611912e3c51c91"

Reply:
Resolved